packages/gateway/server.mjs

```diff
@@ -22,9 +22,7 @@ import socketIo from './rest/socket-io.mjs';
  * @property {string} TIMELD_GATEWAY_SMTP__AUTH__USER
  * @property {string} TIMELD_GATEWAY_SMTP__AUTH__PASS
  * @property {string} [TIMELD_GATEWAY_ADDRESS__PORT] defaults to 8080
- * @property {string} [TIMELD_GATEWAY_ADDRESS__HOST]
- * @property {string} [TIMELD_GATEWAY_ADDRESS__PATH]
- * @see https://nodejs.org/docs/latest-v16.x/api/net.html#serverlistenoptions-callback
+ * @property {string} [TIMELD_GATEWAY_ADDRESS__HOST] defaults to any-network-host
```

Comment:
What's the rationale behind removing `TIMELD_GATEWAY_ADDRESS__HOST` and `TIMELD_GATEWAY_ADDRESS__PATH`?  Will this not result in issues with the server listening on IPv6 by default, which isn't supported in a Docker environment?

Reply:
It's still there, just has a comment now, see last line (25) in snippet